### PR TITLE
chore: move channels into impl classes

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: src/Playwright/Transport/ChannelOwnerBase.cs [hunk 1]
```diff
@@ -136,4 +136,15 @@ private void UpdateEventSubscription(string eventName, bool enabled)
             }),
             true).IgnoreException();
     }
+
+    internal Task<JsonElement?> SendMessageToServerAsync(
```
Comment (yury-s) [inline src/Playwright/Transport/ChannelOwnerBase.cs:140]: why do we need this specialization of the template?

**Review by yury-s — APPROVED** [review]

File: src/Playwright/Transport/ChannelOwnerBase.cs (hunk 1 above)
Comment (mxschmitt) [reply src/Playwright/Transport/ChannelOwnerBase.cs:140]: Some legacy, let me try to follow-up with that.

Comment (mxschmitt) [reply src/Playwright/Transport/ChannelOwnerBase.cs:140]: Turns out we have it twice:

a) SendMessageToServerAsync without the generic gives you a JsonElement? back.
b) SendMessageToServerAsync with a generic which internally deserialises the JSON for you into some specific data structure T

So no need to follow-up!

**Comment by maw136** [comment]
This broke implementation:
` { "chromiumSandbox", options.ChromiumSandbox },
                { "slowMo", options.ChromiumSandbox },`

**Pull request merged**